Lib/platform.py

```diff
@@ -614,17 +595,12 @@ def _syscmd_file(target, default=''):
         return default
     target = _follow_symlinks(target)
     try:
-        proc = subprocess.Popen(['file', target],
-                stdout=subprocess.PIPE, stderr=subprocess.STDOUT)
-
-    except (AttributeError, OSError):
+        output = subprocess.check_output(['file', target],
```

Comment:
The code contradicts the docstring. This can lead to returning an incorrect result if the executable path contains some key strings like "'32-bit'" or "PE". This is a separate issue, but it could be better to fix it before merging this PR. Merging this PR will make backporting harder.

Reply:
It has been changed by commit 685fffa8f427b3a768b232170752565d58c32112, https://bugs.python.org/issue16112 The docstring seems to be outdated.

Reply:
> This can lead to returning an incorrect result if the executable path contains some key strings like "'32-bit'" or "PE".

I would prefer to keep this PR simple, and so don't try to fix this bug nor update the docstring.

Reply:
@serhiy-storchaka created https://bugs.python.org/issue35348